scripts/asm-installer/install_asm

```diff
@@ -2394,6 +2393,10 @@ outro() {
     info "${OUTPUT_DIR}/${RAW_YAML}"
     info "The full, expanded set of kubernetes resources can be found at:"
     info "${OUTPUT_DIR}/${EXPANDED_YAML}"
+  else
+    info "You can find the validatingwebhook and gateway configuration files at:"
+    info "${OUTPUT_DIR}/${MANAGED_WEBHOOKS}"
+    info "${OUTPUT_DIR}/managed_control_plane_gateway.yaml"
```

Comment:
We still need to modify the name of the webhook in this file - otherwise when user applies it will still be patched by istiod and stop working.

I guess if we do modify the name - we can as well apply it...